tools/solidityUpgrade/SourceTransform.h

```diff
@@ -56,23 +54,15 @@ class LocationHelper
 class SourceAnalysis
 {
 public:
-	static bool isMultilineKeyword(
-		langutil::SourceLocation const& _location,
-		std::string const& _keyword
-	)
+	static bool
+	isMultilineKeyword(langutil::SourceLocation const& _location, std::string const& _keyword)
```

Comment:
Yeah, I guess there's quite a few issues with it still...

Reply:
This is fixed now by setting the penalty `PenaltyReturnTypeOnItsOwnLine` to a very high number, so `clang-format` chooses to break after the `(`.